app/views/welcome/_nav_bar.html.erb

```diff
@@ -0,0 +1,28 @@
+ <nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <a class="navbar-brand" <%= link_to "Rebuilding Together", root_path %></a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item active">
+        <a class= "nav-link",<%= link_to "Voluntueer", volunteers_path %> <span class="sr-only">(current)</span></a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" <%= link_to "Home Owner", root_path %></a>
+      </li>
+      </div>
+    <div>
+     <ul class="nav navbar-nav navbar-right">
+       <li>
+        <a class= "nav-link" <%= link_to "Sign up", new_user_registration_path %> <span class="sr-only">(current)</span></a>
+       </li>
+        <li>
+        <a class= "nav-link" <%= link_to "Login",  new_user_session_path %> <span class="sr-only">(current)</span></a>
+       </li>
```

Comment:
Please just use `link_to` for these two links (following the format suggested above).